cli/src/main/scala/org/apache/celeborn/cli/master/MasterOptions.scala

```diff
@@ -120,4 +120,46 @@ final class MasterOptions {
     names = Array("--delete-apps"),
     description = Array("Delete resource of an application."))
   private[master] var deleteApps: Boolean = _
+
```

Comment:
There are two more commands to implement.
- show raft group info 
- print local raft meta conf